website/docs/user_guide/environments.md

```diff
@@ -56,6 +56,27 @@ In order for the SDK to download the feature toggle configuration for the correc
 
 ![An API key creation form. The form's fields are "username", "token type", "project", and, crucially, "environment". The development environment is selected.](/img/environments_api_keys.png 'Create Environment specific API Keys')
 
+## Cloning environments
+
+:::note Availability
+
+Environment cloning is an upcoming feature, and is scheduled to become available in one of the next few releases.
+
+:::
+
+Unleash environments can be cloned. Cloning an environment creates a **new** environment based on the selected source environment. When cloning an environment, you select any number of projects whose feature toggle configurations will also be cloned. These projects will have identical configurations for the source and target environments. However, the environments can then be configured independently and will not stay in sync with each other.
+
+When cloning an environment, you must give the new environment
+- a name
+- an environment type
+- a list of projects to clone configurations in
```

Comment:
Hmm, maybe this could be a little clearer. It's not obvious when it just says "configurations", I think 😅 

```suggestion
- a list of projects to clone feature configurations in
```